gdctx/global.go

```diff
@@ -37,6 +38,7 @@ var (
 	OpVersion      int
 	EtcdProcessCtx *os.Process
 	EtcdClient     etcdclient.Client
+	EtcdInstance   *embed.Etcd
```

Comment:
I expect it to be referenced elsewhere in code too (just in my head right now), especially around add and remove peer operations. I'll move it out later if needed.